XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs

```diff
@@ -42,6 +47,78 @@ private void InitializeData()
             BarItems.Add(new Bar(300, this.MaxLength));
             BarItems.Add(new Bar(25, this.MaxLength));
             BarItems.Add(new Bar(175, this.MaxLength));
+
+            ObservableCollection<object> basicData = new ObservableCollection<object>();
+            basicData.Add(64);
+            basicData.Add("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.");
+            basicData.Add(128);
+            basicData.Add("Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.");
+            basicData.Add(256);
+            basicData.Add("Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.");
+            basicData.Add(512);
+            basicData.Add("Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.");
+            basicData.Add(1024);
+            MixedTypeRepeater.ItemsSource = basicData;
+
+            ObservableCollection<NestedCategory> nestedCategories = new ObservableCollection<NestedCategory>();
```

Comment:
NIT: Do these need to be `ObservableCollection`? Probably not, since the collections aren't changing during runtime.

Reply:
Changed to `List`.